modules/shortcodes/js/recipes-printthis.js

```diff
@@ -35,255 +35,260 @@
  *
  * Notes:
  *  - the loadCSS will load additional css (with or without @media print) into the iframe, adjusting layout
- * 
```

Comment:
@sirreal  
This seems to be a jquery plugin as well. 
That could also be updated to a newer version see 
https://github.com/jasonday/printThis/blob/master/printThis.js

Reply:
With the source duplicated here, I don't see much reason not to run prettier on this.

Updating the plugin would be material for another PR.